AEPMessaging/Sources/SharedStateResult+Messaging.swift

```diff
@@ -0,0 +1,26 @@
+/*
+ Copyright 2021 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import AEPCore
+import Foundation
+
+extension SharedStateResult {
+    var experienceEventDataset: String? {
+        return value?[MessagingConstants.SharedState.Configuration.EXPERIENCE_EVENT_DATASET] as? String
+    }
+    
+    var pushPlatform: String {
```

Comment:
to me the pushPlatform name seems confusing
can we name to something different here like `pushMode` or `pushServer`